service.go

```diff
@@ -218,6 +219,15 @@ func init() {
 			desc:   "change your password",
 			handle: handlePasswordChange,
 		},
+		"channel": {
+			children: serviceCommandSet{
+				"update": {
+					usage:  "<name> [-auto-detach duration] [-detach-relay none|highlight|message] [-reattach none|highlight|message]",
```

Comment:
* In the future maybe we'll want to set a filter for auto-detach, e.g. auto-detach if no highlight for a while.
* Maybe `-auto-detach-after` would be a better name?
* Maybe `-reattach-on` would be a better name?

Reply:
`-reattach-on` is nice. what about `-detach-after`?

Reply:
Works for me